app/src/main/java/ch/epfl/culturequest/ui/map/MapsFragment.java

```diff
@@ -121,6 +195,28 @@ public void onResume() {
         if (mMap != null) {
             getLocationPermission();
         }
+        getProfilePicture();
+    }
+
+    private void getProfilePicture(){
+        Picasso.get().load(Profile.getActiveProfile().getProfilePicture()).into(new Target() {
+            @Override
+            public void onBitmapLoaded(Bitmap bitmap, Picasso.LoadedFrom from) {
+                Log.i("PICTURE", "PICTURE LOADED");
+                profilePicture = getCircularBitmap(bitmap);
```

Comment:
Do you still need these logs or you used it for debug ?

Reply:
Yeah, I can actually change that, thanks !